src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -243,6 +243,8 @@ def _print_examples(help_file):
         indent = 2
         _print_indent('{0}'.format(e.text), indent)
 
+        print('')
```

Comment:
Super nit-picky but this would leave an extra newline at the end of the last help example.
If it doesn't look odd when it's displayed, it's fine, if not, we should change.